src/FPIndia/View.hs

```diff
@@ -53,12 +53,15 @@ renderBody rp model r = do
         renderMarkdown model "resources.md"
     renderFooter
 
-renderNavbar :: Prism' FilePath Route -> H.Html
-renderNavbar rp =
+renderNavbar :: Prism' FilePath Route -> HtmlRoute -> H.Html
+renderNavbar rp r =
   H.nav ! A.class_ "w-full h-1/4 text-xl font-bold flex space-x-4  mb-4" $ do
-    forM_ universe $ \r -> renderURL (H.toHtml $ routeTitle r) (routeHref rp r)
+    forM_ universe $ \x -> if x == r then renderURL (H.toHtml $ routeTitle x) (routeHref rp x) True else renderURL (H.toHtml $ routeTitle x) (routeHref rp x) False
   where
-    renderURL menuItem path = H.a ! A.href path ! A.class_ "p-2 border-b-2 transition color hover:border-stone-900 duration-300" $ menuItem
+    renderURL menuItem path flag =
+      if flag
+        then H.a ! A.href path ! A.class_ "p-2 border-b-2 border-stone-900" $ menuItem
+        else H.a ! A.href path ! A.class_ "p-2 border-b-2 transition color hover:border-stone-900 duration-300" $ menuItem
```

Comment:
The conditionally is for the class string only, not the entire Html widget. So you want to move the `if` as close to inside as possible. Maybe use `let` block to create a `linkClass` text variable and assign that conditionally.